Version_Control_Exercise.md

```diff
@@ -0,0 +1,183 @@
+# Version Control Exercise
+Note: This tutorial is based on the EECS 280 Git Tutorial from Lab 0. All credit goes to James Juett and Zachary Goldston for designing the original lab.
+
+Before beginning this exercise, we recommend going through the main README in the `workflow-intros` repository, to get some basic exposure to git. 
+
+For the purposes of these exercises, we recommend you work in groups of 2-3 to go through this tutorial. This way, you and your group will gain a basic understanding of Git Workflow and handling code changes across different remotes/computers. For the purposes of this tutorial, we will assume a group of 3, but feel free to adjust roles to the number of people in your group.
+
+## Initial Steps
+
+All group members should verify `git` is properly installed via the command `git --version` in your terminal. Additionally, all group members should configure their git profile, if not done yet. You can do this by running the two commands:
+
+    git config --global user.name "username"
+    git config --global user.email "youremail"
+
+(Replace the `username` and `youremail` parts with the GitHub username and email you plan to use when working on MRover! Note that we do not use Michigan's GitHub server)
+
+Everyone in the group should sign into their GitHub account they plan to use with the MRover team, if they have not done already.
+
+**Person 1** should create a new repository on GitHub.  From here, Person 1 should ensure the following when creating the new project:
+* Name it `mrover-vc-exercise`
```

Comment:
Maybe it would be good to explain the 'vc' acronym, people may not realize what this is. Within this, could say that svn is an alternative to git

Reply:
Alternatively I guess I could talk about this in the workflow-intros readme, but I think it would make a little more sense here? That other one is really just about pull requests

Reply:
I'll put something in the beginning that talks about VCS and Git vs. SVM; I didn't feel like it would be necessary, but it would be good for members to know

Reply:
sounds good